services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

```diff
@@ -799,6 +799,35 @@ paths:
               schema:
                 title: Response Get Service Resources
                 type: object
+  /v0/catalog/services/{service_key}/{service_version}/pricing-plan:
```

Comment:
I was wondering whether we should include `/v0/credits-price` somewhere within these pricing-plans etc

Reply:
Yes we can discuss in person where we will add it :+1: